img/config.md
```javascript
const config = {
  //网页元数据
  metaData:{
    title: '昼有昨夕的芳的个人主页🎉',
    description: '欢迎来到我的奇妙世界！',
    keywords: '个人主页,个人网站',
    icon: '/favicon.ico'   //网页图标，支持外链
  },

  avatar: "/img/avatar.jpg", // 头像
  welcometitle: "Ciallo～(∠・ω< )⌒☆", // 标题

  // 颜色配置
  color: {
    themecolor: "#FFFFFF", // 主题颜色，推荐趋于亮白可带有轻微色调，例： #D1FFEC
    welcometitlecolor: "#FFFFFF", // 标题颜色 例： #7BFFC9
    turntablecolor1: "#FFFF00",  // 转盘渐变色一
    turntablecolor2: "#00FFFF"   // 转盘渐变色二
  },
  
  brightness: 80, // 背景亮度 --%
  blur: 10, // 毛玻璃模糊效果

  // 我的标签
  tags : ['乐观开朗','温柔体贴','随和亲切','冷静沉着','才思敏捷','风趣幽默','刚正不阿','善解人意'],

  // 默认背景壁纸
  background: {
    "pc":{   //pc端
      "type":"pic",   //"pic":静态壁纸;"video":动态壁纸
      "datainfo":{
        "title":"随机壁纸", 
        "preview":"https://t.mwm.moe/pc",
        "url":"https://t.mwm.moe/pc",     //当然，也可填写网络地址或壁纸api，如随机PC壁纸api："url":"https://t.mwm.moe/pc"
      },
    },
    "mobile":{   //移动端
      "type":"pic",
      "datainfo":{
        "title":"0001", 
        "preview":"https://t.mwm.moe/mp",
        "url":"https://t.mwm.moe/mp"  //同理，随机移动端壁纸："url":"https://t.mwm.moe/mp"
      }
    }
      
  },

  //极坐标图数据
  polarChart:{
    skills: ['动画', '轻小说', '手游', '端游', 'GalGame', '文学'],
    skillPoints: [91, 78, 70, 67, 81, 61],
  },

  //社交按钮，图标使用mdi图标（https://pictogrammers.com/library/mdi/），复制图标名称即可
  socialPlatformIcons: [
    {icon:"mdi-github",link:"https://github.com/zfyx-ice"},
    {icon:"mdi-email",link:"mailto:wjhyuki_0201@163.com"},
    {icon:"mdi-qqchat",link:"https://im.qq.com/"},
    {icon:"mdi-wechat",link:"https://wx.qq.com/"},
    {icon:"mdi-youtube",link:"https://www.youtube.com"},
    {icon:"mdi-facebook",link:"https://www.facebook.com"}
  ],

  //打字机
  typeWriterStrings: [
    "如果你看到了这行字，说明我已经成功吸引到了你的注意力。",
    "若是你累了，不妨停下来，看看这世间的风景。"
    "顶峰的少年，给了你所有细节，你却说我不是迪迦，给不了你想要的光。",
    "心简单，世界就简单，幸福才会生长；心自由，生活就自由，到哪都有快乐。",
    "生命太短，没有时间留给遗憾，若不是终点，请微笑一直向前。"
  ],

  //音乐播放配置，采用MetingJS Api(https://github.com/metowolf/MetingJS)
  musicPlayer:{
    server: 'netease',  //服务提供商 --网易云音乐
    type: 'playlist',   //歌单类型
    id: '2028178887'  //歌单id ---> music.163.com/#/playlist?id=2028178887
  },

  //壁纸数据 -----可以将壁纸文件上传到图床获取网络直链。若想调用api，请前往脚本自行修改逻辑
  wallpaper:{
    pic:[
      {"title":"海洋女孩","preview":"/img/wallpaper/static/海洋女孩/image-pre.webp","url":"/img/wallpaper/static/海洋女孩/image.png"},
      {"title":"书房夜晚","preview":"/img/wallpaper/static/书房夜晚/image-pre.webp","url":"/img/wallpaper/static/书房夜晚/image.png"},
      {"title":"安逸舒适","preview":"/img/wallpaper/static/安逸舒适/image-pre.webp","url":"/img/wallpaper/static/安逸舒适/image.png"},
    ],
    picMobile:[
      {"title":"0001","preview":"/img/wallpaper/static-mobile/0001/image-pre.webp","url":"/img/wallpaper/static-mobile/0001/image.png"},
      {"title":"0002","preview":"/img/wallpaper/static-mobile/0002/image-pre.webp","url":"/img/wallpaper/static-mobile/0002/image.png"},
      {"title":"0003","preview":"/img/wallpaper/static-mobile/0003/image-pre.webp","url":"/img/wallpaper/static-mobile/0003/image.png"},
      {"title":"0004","preview":"/img/wallpaper/static-mobile/0004/image-pre.webp","url":"/img/wallpaper/static-mobile/0004/image.png"},
    ],
    video:[
      {
        "title":"尼尔：机械纪元 团队",
        "preview":"/img/wallpaper/dynamic/尼尔：机械纪元 团队/Nier-Automata-Team-pre.webm",
        "url":"/img/wallpaper/dynamic/尼尔：机械纪元 团队/Nier-Automata-Team.webm"
      },
      {
        "title":"向往航天的女孩",
        "preview":"/img/wallpaper/dynamic/向往航天的女孩/Toy-Aeroplane-pre.webm",
        "url":"/img/wallpaper/dynamic/向往航天的女孩/Toy-Aeroplane.webm"
      },
    ],
    videoMobile:[
      {
        "title":"幻觉镇-gaako_illust",
        "preview":"/img/wallpaper/dynamic-mobile/幻觉镇-gaako_illust/Hallucination_town-pre.mp4",
        "url":"/img/wallpaper/dynamic-mobile/幻觉镇-gaako_illust/Hallucination_town.mp4"
      },
      {
        "title":"chuva",
        "preview":"/img/wallpaper/dynamic-mobile/chuva/chuva-pre.mp4",
        "url":"/img/wallpaper/dynamic-mobile/chuva/chuva.mp4"
      },
      {
        "title":"Doodle-小猫女仆降临",
        "preview":"/img/wallpaper/dynamic-mobile/Doodle-小猫女仆降临/d12-pre.mp4",
        "url":"/img/wallpaper/dynamic-mobile/Doodle-小猫女仆降临/d12.mp4"
      },
    ],
  },

  //项目卡片 其中 字段"show"控制初始卡片的text是否展开
  projectcards:[ 
    {go:"🚀 前往",img:"/img/sunshine.jpg",title:"我的博客",subtitle:"没什么可看的", text:"平时会写写自己生活中的感受.",url:"https://zfyx-ice.github.io/",show:false},
    {go:"🗂️ 前往",img:"/img/sunshine.jpg",title:"Project 2",subtitle:"2,000 miles of wonder", text:"If you see this line, I've managed to get your attention.",url:"https://leleo.top",show:false},
    {go:"📝 前往",img:"/img/sunshine.jpg",title:"Project 3",subtitle:"3,000 miles of wonder", text:"If you see this line, I've managed to get your attention.",url:"https://leleo.top",show:false},
    {go:"👍 前往",img:"/img/sunshine.jpg",title:"Project 4",subtitle:"4,000 miles of wonder", text:"If you see this line, I've managed to get your attention.",url:"https://leleo.top",show:false},
    {go:"🗃 前往",img:"/img/sunshine.jpg",title:"Project 5",subtitle:"5,000 miles of wonder", text:"If you see this line, I've managed to get your attention.",url:"https://leleo.top",show:false},
    {go:"🎨 前往",img:"/img/sunshine.jpg",title:"Project 6",subtitle:"6,000 miles of wonder", text:"If you see this line, I've managed to get your attention.",url:"https://leleo.top",show:false},
    {go:"💍 前往",img:"/img/sunshine.jpg",title:"Project 7",subtitle:"7,000 miles of wonder", text:"If you see this line, I've managed to get your attention.",url:"https://leleo.top",show:false},
    {go:"🔍 前往",img:"/img/sunshine.jpg",title:"Project 8",subtitle:"8,000 miles of wonder", text:"If you see this line, I've managed to get your attention.",url:"https://leleo.top",show:false},
  ],
  
  statement: ["备案号：泷原ICP备114514号","Copyright © 2025 Yuki"],
}

export default config
```


src/config.js
const config = {
  metaData: {
    title: '昼有昨夕的芳的个人主页🎉',
    description: '欢迎来到我的奇妙世界！',
    keywords: '个人主页,个人网站',
    icon: '/favicon.ico'
  },
  avatar: "/img/avatar.jpg",
  welcometitle: "Ciallo～(∠・ω< )⌒☆",
  color: {
    themecolor: "#FFFFFF",
    welcometitlecolor: "#FFFFFF",
    turntablecolor1: "#FFFF00",
    turntablecolor2: "#00FFFF"
  },
  brightness: 80,
  blur: 10,
  tags: ['乐观开朗', '温柔体贴', '随和亲切', '冷静沉着', '才思敏捷', '风趣幽默', '刚正不阿', '善解人意'],
  background: {
    "pc": {
      "type": "pic",
      "datainfo": {
        "title": "随机壁纸",
        "preview": "https://t.mwm.moe/pc",
        "url": "https://t.mwm.moe/pc"
      }
    },
    "mobile": {
      "type": "pic",
      "datainfo": {
        "title": "0001",
        "preview": "https://t.mwm.moe/mp",
        "url": "https://t.mwm.moe/mp"
      }
    }
  },
  polarChart: {
    skills: ['动画', '轻小说', '手游', '端游', 'GalGame', '文学'],
    skillPoints: [91, 78, 70, 67, 81, 61]
  },
  socialPlatformIcons: [
    { icon: "mdi-github", link: "https://github.com/zfyx-ice" },
    { icon: "mdi-email", link: "mailto:wjhyuki_0201@163.com" },
    { icon: "mdi-qqchat", link: "https://im.qq.com/" },
    { icon: "mdi-wechat", link: "https://wx.qq.com/" },
    { icon: "mdi-youtube", link: "https://www.youtube.com" },
    { icon: "mdi-facebook", link: "https://www.facebook.com" }
  ],
  typeWriterStrings: [
    "如果你看到了这行字，说明我已经成功吸引到了你的注意力。",
    "若是你累了，不妨停下来，看看这世间的风景。",
    "顶峰的少年，给了你所有细节，你却说我不是迪迦，给不了你想要的光。",
    "心简单，世界就简单，幸福才会生长；心自由，生活就自由，到哪都有快乐。",
    "生命太短，没有时间留给遗憾，若不是终点，请微笑一直向前。"
  ],
  musicPlayer: {
    server: 'netease',
    type: 'playlist',
    id: '2028178887'
  },
  wallpaper: {
    pic: [
      { "title": "海洋女孩", "preview": "/img/wallpaper/static/海洋女孩/image-pre.webp", "url": "/img/wallpaper/static/海洋女孩/image.png" },
      { "title": "书房夜晚", "preview": "/img/wallpaper/static/书房夜晚/image-pre.webp", "url": "/img/wallpaper/static/书房夜晚/image.png" },
      { "title": "安逸舒适", "preview": "/img/wallpaper/static/安逸舒适/image-pre.webp", "url": "/img/wallpaper/static/安逸舒适/image.png" }
    ],
    picMobile: [
      { "title": "0001", "preview": "/img/wallpaper/static-mobile/0001/image-pre.webp", "url": "/img/wallpaper/static-mobile/0001/image.png" },
      { "title": "0002", "preview": "/img/wallpaper/static-mobile/0002/image-pre.webp", "url": "/img/wallpaper/static-mobile/0002/image.png" },
      { "title": "0003", "preview": "/img/wallpaper/static-mobile/0003/image-pre.webp", "url": "/img/wallpaper/static-mobile/0003/image.png" },
      { "title": "0004", "preview": "/img/wallpaper/static-mobile/0004/image-pre.webp", "url": "/img/wallpaper/static-mobile/0004/image.png" }
    ],
    video: [
      { "title": "尼尔：机械纪元 团队", "preview": "/img/wallpaper/dynamic/尼尔：机械纪元 团队/Nier-Automata-Team-pre.webm", "url": "/img/wallpaper/dynamic/尼尔：机械纪元 团队/Nier-Automata-Team.webm" },
      { "title": "向往航天的女孩", "preview": "/img/wallpaper/dynamic/向往航天的女孩/Toy-Aeroplane-pre.webm", "url": "/img/wallpaper/dynamic/向往航天的女孩/Toy-Aeroplane.webm" }
    ],
    videoMobile: [
      { "title": "幻觉镇-gaako_illust", "preview": "/img/wallpaper/dynamic-mobile/幻觉镇-gaako_illust/Hallucination_town-pre.mp4", "url": "/img/wallpaper/dynamic-mobile/幻觉镇-gaako_illust/Hallucination_town.mp4" },
      { "title": "chuva", "preview": "/img/wallpaper/dynamic-mobile/chuva/chuva-pre.mp4", "url": "/img/wallpaper/dynamic-mobile/chuva/chuva.mp4" },
      { "title": "Doodle-小猫女仆降临", "preview": "/img/wallpaper/dynamic-mobile/Doodle-小猫女仆降临/d12-pre.mp4", "url": "/img/wallpaper/dynamic-mobile/Doodle-小猫女仆降临/d12.mp4" }
    ]
  },
  projectcards: [
    { "go": "🚀 前往", "img": "/img/sunshine.jpg", "title": "我的博客", "subtitle": "没什么可看的", "text": "平时会写写自己生活中的感受.", "url": "https://zfyx-ice.github.io/", "show": false },
    { "go": "🗂️ 前往", "img": "/img/sunshine.jpg", "title": "Project 2", "subtitle": "2,000 miles of wonder", "text": "If you see this line, I've managed to get your attention.", "url": "https://leleo.top", "show": false },
    { "go": "📝 前往", "img": "/img/sunshine.jpg", "title": "Project 3", "subtitle": "3,000 miles of wonder", "text": "If you see this line, I've managed to get your attention.", "url": "https://leleo.top", "show": false },
    { "go": "👍 前往", "img": "/img/sunshine.jpg", "title": "Project 4", "subtitle": "4,000 miles of wonder", "text": "If you see this line, I've managed to get your attention.", "url": "https://leleo.top", "show": false },
    { "go": "🗃 前往", "img": "/img/sunshine.jpg", "title": "Project 5", "subtitle": "5,000 miles of wonder", "text": "If you see this line,I've managed to get your attention.", "url": "https://leleo.top", "show": false }
    ],
}
